client/matching/loadbalancer_test.go

```diff
@@ -138,8 +145,14 @@ func Test_defaultLoadBalancer_PickWritePartition(t *testing.T) {
 
 			for i := 0; i < 100; i++ {
 				got := lb.PickWritePartition(tt.args.domainID, tt.args.taskList, tt.args.taskListType, tt.args.forwardedFrom)
-				str := strconv.Itoa(tt.expectedMax)
-				assert.Contains(t, got, str)
+				found := false
+				for _, val := range tt.expectedVal {
+					if strings.Contains(val, got) {
+						found = true
+						break
+					}
+				}
+				assert.True(t, found)
```

Comment:
These can be simplified by
```suggestion
				assert.Contains(t, tt.expectedVal, got)
```